core/src/main/scala/kafka/log/Log.scala

```diff
@@ -1970,7 +1969,6 @@ class Log(@volatile private var _dir: File,
           baseOffset = newOffset,
           config,
           time = time,
```

Comment:
there is a same issue https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/log/LogCleaner.scala#L456

Reply:
Done.